Applications/src/detect-cardiac-phases.cc

```diff
@@ -29,15 +29,18 @@ using namespace mirtk;
 void PrintHelp(const char *name)
 {
   cout << endl;
-  cout << "Usage: " << name << " <cine> <ED output> <ES output>" << endl;
+  cout << "Usage: " << name << " <cine> --output-ed <ED output> --output-es <ES output>" << endl;
   cout << endl;
   cout << "Description:" << endl;
   cout << "  This program detects ED and ES phases from a cine image sets." << endl;
   cout << endl;
   cout << "Arguments:" << endl;
-  cout << "  cine        Input cine image sets." << endl;
-  cout << "  ED output   ED phase image." << endl;
-  cout << "  ES output   ES phase image." << endl;
+  cout << "  cine         Input cine image sets." << endl;
+  cout << "Optional arguments:" << endl;
+  cout << "  --output-ed  ED output   ED phase image." << endl;
```

Comment:
This help is a bit confusing. It's more custom not to use spaces in the argument variable name.

```
  --output-ed path   Output file path of ED phase image.
  --output-es path   Output file path of ES phase image.
```